e2e/components/sidenav-e2e.spec.ts

```diff
@@ -0,0 +1,28 @@
+import {browser, by, element, protractor} from 'protractor';
+const EC = protractor.ExpectedConditions;
+
+
+describe('sidenav', () => {
```

Comment:
nit: to match style used elsewhere in the project, indent size should be 2 spaces